### PR TITLE
Create Priyanshul.json

### PR DESCRIPTION
**Pull request by NerdIsAwake**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview
priyanshul130.github.io

### CODE REVIEW

File: domains/Priyanshul.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My Portfolio Website",
+    "repo": "https://github.com/Priyanshul1307/Priyanshul1307.github.io",
+    "owner": {
+      "username": "Priyanshul1307",
+      "email": "piyanshul1307@gmail.com"
+    },
+    "record": {
+      "CNAME": "Priyanshul1307.github.io"
```
Comment (CuteDog5695) [inline domains/Priyanshul.json:9]: Site doesn't seem to be accessible, please give a preview of your website

File: domains/Priyanshul.json
Comment (CuteDog5695) [inline domains/Priyanshul.json:1]: Rename the json file to priyanshul.json please

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/Priyanshul.json (hunk 1 above)
Comment (Priyanshul130) [reply domains/Priyanshul.json]: sorry for the wrong address for the website its Priyanshul130.github.io
and i have changed the file name to priyanshul.json

Comment (Priyanshul130) [reply domains/Priyanshul.json]: https://github.com/user-attachments/assets/9cceb502-9a5f-4490-ad3f-34707133275e

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**